src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java

```diff
@@ -699,9 +702,7 @@ public static String getSinceVersion(String moduleName, DetailNode moduleJavadoc
         if (superClassSinceVersion != null) {
             sinceVersion = superClassSinceVersion;
         }
-        else if (SUPER_CLASS_PROPERTIES_JAVADOCS.containsKey(propertyName)
-                || TOKENS.equals(propertyName)
-                || JAVADOC_TOKENS.equals(propertyName)) {
+        else if (TOKENS.equals(propertyName) || JAVADOC_TOKENS.equals(propertyName)) {
```

Comment:
````suggestion
        else if (TOKENS.equals(propertyName)
                || JAVADOC_TOKENS.equals(propertyName)) {
````

Reply:
Sure sir

Reply:
@rnveach Even After making these changes it is not updating sinceVersion.

Reply:
@Rohanraj123 , restore formatting only
```
else if (TOKENS.equals(propertyName)
                || JAVADOC_TOKENS.equals(propertyName)) {
```

Reply:
code looks good.

Reply:
> Even After making these changes it is not updating sinceVersion.

What do you mean?
It was updated at https://github.com/checkstyle/checkstyle/pull/13945/files#diff-eaff5fa3cb101145757a2bd9ebe2e805e8fd8057ae27f9e21ef91900cb7938cc

Reply:
Okay got it thanks !

Reply:
> @Rohanraj123 , restore formatting only
> 
> ```
> else if (TOKENS.equals(propertyName)
>                 || JAVADOC_TOKENS.equals(propertyName)) {
> ```

did it ! @romani Sir